mcc/odin/odin/snapshot_list_test.go

```diff
@@ -17,3 +21,39 @@ var exampleSnapshot1 = &rds.DBSnapshot{
 	MasterUsername:       aws.String("owner"),
 	Status:               aws.String("available"),
 }
+
+type listSnapshotsCase struct {
+	testCase
+	name       string
+	snapshots  []*rds.DBSnapshot
+	instanceID string
+}
+
+var listSnapshotsCases = []listSnapshotsCase{
+	// No snapshots for any instance
+	{
+		testCase: testCase{
+			expected:      []*rds.DBSnapshot{},
+			expectedError: "",
+		},
+		name:       "No snapshots for any instance",
+		snapshots:  []*rds.DBSnapshot{},
+		instanceID: "",
+	},
+}
+
+func TestListSnapshots(t *testing.T) {
+	svc := newMockRDSClient()
+	for _, test := range listSnapshotsCases {
```

Comment:
Simple for each test case. The `_` makes the index being returned by `range` not to be taken into account.